### PR TITLE
updated admob, facebook, chartboost and supersonic ad adapters

### PR DESCRIPTION
**Pull request by vivekb91**

### CODE REVIEW

File: build/providers/supersonic/android/files/SupersonicPlugin.java [hunk 1]
```diff
@@ -327,42 +381,52 @@ public void onCreate(Activity activity, Bundle savedInstanceState) {
     if(listener == null) {
       listener = new SupersonicListener();
     }
+        
     //Initialize the SDK, passing the current context to the method
-    mSupersonicInstance = SupersonicFactory.getInstance();
+    //mSupersonicInstance = IronSource.getInstance();
```
Comment (rampr) [inline build/providers/supersonic/android/files/SupersonicPlugin.java]: this is not needed ? can we remove it ?

File: build/providers/supersonic/android/files/SupersonicPlugin.java [hunk 2]
```diff
@@ -327,42 +381,52 @@ public void onCreate(Activity activity, Bundle savedInstanceState) {
...
   }
 
   public void initInterstitial(String jsonData) {
     logger.log("{supersonic} Init Interstitial");
 
-    if(mSupersonicInstance != null) {
-      mSupersonicInstance.setInterstitialListener(listener);
-      mSupersonicInstance.initInterstitial(_activity, appKey, getUserId(jsonData));
-    }
+    //if(mSupersonicInstance != null) {
```
Comment (rampr) [inline build/providers/supersonic/android/files/SupersonicPlugin.java]: remove commented out code

**Comment by rampr** [comment]
should we also rename the repo ?

**Comment by rampr** [comment]
let's review it here and move it to the new repo once review is complete. Makes sense ?

File: build/providers/supersonic/android/files/SupersonicPlugin.java [hunk 3]
```diff
@@ -327,42 +381,52 @@ public void onCreate(Activity activity, Bundle savedInstanceState) {
...
   }
 
   public void initVideoAd(String jsonData) {
     logger.log("{supersonic} Init video Ad");
 
-    if(mSupersonicInstance != null) {
-      mSupersonicInstance.setRewardedVideoListener(listener);
-      mSupersonicInstance.initRewardedVideo(_activity, appKey, getUserId(jsonData));
-    }
+    //if(mSupersonicInstance != null) {
```
Comment (rampr) [inline build/providers/supersonic/android/files/SupersonicPlugin.java]: remove commented out code

File: build/providers/supersonic/android/files/SupersonicPlugin.java [hunk 4]
```diff
@@ -327,42 +381,52 @@ public void onCreate(Activity activity, Bundle savedInstanceState) {
...
   }
 
   public void initOfferWallAd(String jsonData) {
     logger.log("{supersonic} Init offerwall Ad");
 
-    if(mSupersonicInstance != null) {
-      mSupersonicInstance.setOfferwallListener(listener);
-      mSupersonicInstance.initOfferwall(_activity, appKey, getUserId(jsonData));
-    }
+    //if(mSupersonicInstance != null) {
```
Comment (rampr) [inline build/providers/supersonic/android/files/SupersonicPlugin.java]: commented out code

File: build/providers/supersonic/android/files/SupersonicPlugin.java [hunk 5]
```diff
@@ -371,31 +435,33 @@ public void showInterstitial(String jsonData) {
...
+      IronSource.showRewardedVideo();
     }
+    else logger.log("{supersonic} not available");
   }
 
   public void onResume() {
-    if (mSupersonicInstance != null) {
-      mSupersonicInstance.onResume(_activity);
-    }
+    //if (mSupersonicInstance != null) {
```
Comment (rampr) [inline build/providers/supersonic/android/files/SupersonicPlugin.java]: commented out code

File: build/providers/supersonic/android/files/SupersonicPlugin.java [hunk 6]
```diff
@@ -408,9 +474,9 @@ public void onFirstRun() {
   }
 
   public void onPause() {
-    if (mSupersonicInstance != null) {
-      mSupersonicInstance.onPause(_activity);
-    }
+    //if (mSupersonicInstance != null) {
```
Comment (rampr) [inline build/providers/supersonic/android/files/SupersonicPlugin.java]: commented out code can be removed

Comment (bijoshtj) [reply build/providers/supersonic/android/files/SupersonicPlugin.java]: +1

**Review by bijoshtj — APPROVED** [review]

File: build/providers/supersonic/android/files/SupersonicPlugin.java (hunk 1 above)
Comment (vivekb91) [reply build/providers/supersonic/android/files/SupersonicPlugin.java]: yes

**Comment by rampr** [comment]
Closing it as we don't need it anymore. we have moved to ironsource at https://github.com/hashcube/ironsource

**Pull request closed without merging**